pkg/deploymanager/config.go

```diff
@@ -0,0 +1,133 @@
+package deploymanager
+
+import (
+	"fmt"
+	"os"
+
+	olmclient "github.com/operator-framework/operator-lifecycle-manager/pkg/api/client/clientset/versioned"
+	odfv1alpha1 "github.com/red-hat-data-services/odf-operator/api/v1alpha1"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/client-go/tools/clientcmd"
+
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/client-go/rest"
+	crclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// InstallNamespace is the namespace odf is installed into
+const InstallNamespace = "openshift-storage"
+
+//nolint:errcheck // ignoring err check as causing failures
+func init() {
+	odfv1alpha1.SchemeBuilder.AddToScheme(scheme.Scheme)
+}
+
+type DeployManager struct {
+	olmClient      *olmclient.Clientset
```

Comment:
lets use one client for all of the schemes.

Reply:
Having some doubts & issues here as these clients are used separately inside the subscription.go file.